Fix build with GCC 4.4

Lubomir Rintel (Fedora Astronomy) <lkundrak@v3.sk>

diff -up celestia-1.5.0/src/celengine/overlay.h.gcc44 celestia-1.5.0/src/celengine/overlay.h
--- celestia-1.5.0/src/celengine/overlay.h.gcc44	2009-03-01 15:23:44.000000000 +0100
+++ celestia-1.5.0/src/celengine/overlay.h	2009-03-01 15:23:50.000000000 +0100
@@ -12,6 +12,7 @@
 
 #include <string>
 #include <iostream>
+#include <cstdio>
 #include <celtxf/texturefont.h>
 
 
diff -up celestia-1.5.0/src/celengine/star.cpp.gcc44 celestia-1.5.0/src/celengine/star.cpp
--- celestia-1.5.0/src/celengine/star.cpp.gcc44	2009-03-01 15:26:57.000000000 +0100
+++ celestia-1.5.0/src/celengine/star.cpp	2009-03-01 15:27:22.000000000 +0100
@@ -9,6 +9,7 @@
 
 #include <celmath/mathlib.h>
 #include <cstring>
+#include <cstdio>
 #include <cassert>
 #include "celestia.h"
 #include "astro.h"
